Spec.md

```diff
@@ -1,124 +1,136 @@
 # Fernet Spec
 
-This document describes version 0x80 (currently the only
-version) of the fernet format.
+This document describes both version 0x80 (the first version) and 0x81 (the
+second version) of the fernet format. Differences between each variant is
+explained inline.
 
-Conceptually, fernet takes a user-provided *message* (an arbitrary
-sequence of bytes), a *key* (256 bits), and the current
-time, and produces a *token*, which contains the message in a form
-that can't be read or altered without the key.
+Conceptually, fernet takes a user-provided *message* (an arbitrary sequence of
+bytes), a *key* (256 bits), and the current time, and produces a *token*, which
+contains the message in a form that can't be read or altered without the key.
 
-To facilitate convenient interoperability, this spec defines the
-external format of both tokens and keys.
+To facilitate convenient interoperability, this spec defines the external
+format of both tokens and keys.
 
 All encryption in this version is done with AES 128 in CBC mode.
 
-All base 64 encoding is done with the "URL and Filename Safe"
-variant, defined in [RFC 4648](http://tools.ietf.org/html/rfc4648#section-5) as "base64url".
+All base 64 encoding is done with the "URL and Filename Safe" variant, defined
+in [RFC 4648](http://tools.ietf.org/html/rfc4648#section-5) as "base64url".
 
 ## Key Format
 
-A fernet *key* is the base64url encoding of the following
-fields:
+A fernet *key* is the base64url encoding of the following fields:
 
     Signing-key ‖ Encryption-key
 
 - *Signing-key*, 128 bits
+
 - *Encryption-key*, 128 bits
 
 ## Token Format
 
-A fernet *token* is the base64url encoding of the
-concatenation of the following fields:
+A fernet *token* is the base64url encoding of the concatenation of the
+following fields:
 
     Version ‖ Timestamp ‖ IV ‖ Ciphertext ‖ HMAC
 
 - *Version*, 8 bits
+
 - *Timestamp*, 64 bits
+
 - *IV*, 128 bits
+
 - *Ciphertext*, variable length, multiple of 128 bits
+
 - *HMAC*, 256 bits
 
-Fernet tokens are not self-delimiting. It is assumed that the
-transport will provide a means of finding the length of each
-complete fernet token.
+Fernet tokens are not self-delimiting. It is assumed that the transport will
+provide a means of finding the length of each complete fernet token.
 
 ## Token Fields
 
 ### Version
 
-This field denotes which version of the format is being used by
-the token. Currently there is only one version defined, with the
-value 128 (0x80).
+This field denotes which version of the format is being used by the token.
+Currently there is only one version defined, with the value 128 (0x80).
```

Comment:
Nice catch, I totally missed this as I was focused on the timestamp field. Will revise.